### ISSUE TITLE
Test of the new handoff template

### ISSUE THREAD

**Issue by mediajunkie** [department-of-veterans-affairs/va.gov-team#128150]
## Background
- Nothing is being handed off.
- This is just a test of the template

## Goal
- In this case, the goal is to ensure that the ticket meets the needs of UX (@kbermangestring and @conchette ), a11y (@bacitracin ), content (@cinlackey ), and engineering (@SeaJerry , @randimays , and @gracexu94 )
 
### Role doing handoff to eng
 - [ ] Content
 - [ ] Accessibility
 - [ ] UX
 - [x] N/A in this case

### Attachments or artifacts required for handoff:
* _Attach or link to artifacts_
* _Edit/update list as needed_

** Also N/A in this case **

## Process
- [ ] Handoff materials prepared <== found typo, will fix
- [ ] Handoff meeting scheduled
- [ ] Handoff conducted
- [ ] Any follow-up questions addressed
- [ ] Engineering confirms receipt of any needed deliverables
